chemprop/cli/train.py

```diff
@@ -545,15 +541,49 @@ def build_splits(args, format_kwargs, featurization_kwargs):
         **format_kwargs,
         **featurization_kwargs,
     )
-    multicomponent = len(all_data) > 1
 
-    split_kwargs = dict(sizes=args.split_sizes, seed=args.data_seed, num_folds=args.num_folds)
-    split_kwargs["key_index"] = args.split_key_molecule if multicomponent else 0
+    if args.splits_column is not None:
+        df = pd.read_csv(
+            args.data_path, header=None if args.no_header_row else "infer", index_col=False
+        )
+        grouped = df.groupby(df[args.splits_columnsplit_col].str.lower())
```

Comment:
```suggestion
        grouped = df.groupby(df[args.splits_column].str.lower())
```
If you have time, it would be great to have a test for this new CLI (`--splits-column`).

Reply:
I'm glad you mentioned this. I had a test for the old way and have now updated it to work for the new CLI